### PR TITLE
Add feedback notify

### PR DESCRIPTION
**Pull request by punchdrunker**
and fix bug  #304 .

special thanks to @zaki50 san

### CODE REVIEW

**Comment by konifar** [comment]
Awesome! :eyes:

File: app/src/main/java/io/github/droidkaigi/confsched/receiver/SessionScheduleReceiver.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import io.github.droidkaigi.confsched.activity.MainActivity;
 import io.github.droidkaigi.confsched.activity.SessionDetailActivity;
 import io.github.droidkaigi.confsched.model.Session;
+import io.github.droidkaigi.confsched.util.LocaleUtil;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/receiver/SessionScheduleReceiver.java]: Maybe it is not necessary.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -128,4 +128,5 @@
     <string name="session_feedback_submit_failure">Submit failed</string>
     <string name="session_feedback_submit_success">Submit succeeded</string>
     <string name="session_feedback_submit_link">Submit Feedback</string>
+    <string name="finish_notification_title">Please send your feedback for the session.</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Could you create `values-ja/strings.xml` and `values-ar/strings.xml`?

Comment (konifar) [inline app/src/main/res/values/strings.xml]: I guess `session_feedback_notification_title` is better. How do you think?

**Comment by hotchemi** [comment]
Should we close it?

**Comment by konifar** [comment]
Close it.

**Pull request closed without merging**